pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2779,16 +2779,34 @@ cdef class Week(SingleConstructorOffset):
     """
     Weekly offset.
 
+    This can increment the date object by given week or weekdays.
```

Comment:
I think we can remove this (this particular check from `validate_docstrings` is quite far fro being enabled anyway)